### ISSUE TITLE
Feature Request: Focus On Editor

### ISSUE THREAD

**Issue by rbento** [slynch8/10x#2819]
### Key binding to immediately focus on editor

- By pressing ESC, the focus moves to the editor:
    - Set cursor to the last edited location
    - Close the build output window (optionally)
- **HIGHLIGHT:** this is a VERY important workflow feature so one can easily `F7` to build, navigate the output with `F8` and get back to work by pressing `ESC`.
  
- Suggested key bind: ESC

**Comment by slynch8** [comment]
Good idea. For now you can use ctrl+Minus to go back to the last location. Or Alt+Left to move back to the code panel (assuming output panel is in the right column)
